Mathlib/Analysis/SpecialFunctions/Pow/Real.lean

```diff
@@ -764,6 +780,20 @@ theorem exists_rat_pow_btwn {α : Type*} [LinearOrderedField α] [Archimedean α
 
 end Real
 
+namespace Complex
+
+lemma cpow_inv_two_re (x : ℂ) : (x ^ (2⁻¹ : ℂ)).re = sqrt ((abs x + x.re) / 2) := by
+  rw [← ofReal_ofNat, ← ofReal_inv, cpow_ofReal_re, ← div_eq_mul_inv, ← one_div,
+    ← Real.sqrt_eq_rpow, cos_half, ← sqrt_mul, ← mul_div_assoc, mul_add, mul_one, abs_mul_cos_arg]
+  exacts [abs.nonneg _, (neg_pi_lt_arg _).le, arg_le_pi _]
+
+lemma abs_cpow_inv_two_im (x : ℂ) : |(x ^ (2⁻¹ : ℂ)).im| = sqrt ((abs x - x.re) / 2) := by
```

Comment:
is this true as follows?
```suggestion
lemma abs_cpow_inv_two_im (x : ℂ) : (x ^ (2⁻¹ : ℂ)).im = x.im.sign * sqrt ((abs x - x.re) / 2) := by
```

Reply:
No, this isn't true for real negative `x`.

Reply:
How about separate lemmas for `0 ≤ x.im` and `x.im < 0` (in addition to the one you already have)